### PR TITLE
Create Test_file2.cpp

### LINKED ISSUE #2: This is a demo - Need a hello world added
Please add a hello world program

### PR DESCRIPTION
**Pull request by nightflame137**
Fixes #2 

Creates a hello world program that outputs the words "hello world" to a console.

### CODE REVIEW

**Review by KranzTech — CHANGES_REQUESTED** [review]
Please put in at least one line of code

**Comment by nightflame137** [comment]
I added some lines of code, thanks for the suggestion.

File: Test_file2.cpp [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#include <iostream>
```
Comment (KranzTech) [inline Test_file2.cpp:1]: This line of code is great.

**Review by KranzTech — APPROVED** [review]
Approved!  Couldn't be better!

**Pull request merged**